components/03-modules/header/_header.scss

```diff
@@ -1,127 +1,112 @@
-$header__margin                : 4px 0 !default;
-$header__margin--xl            : 0 !default;
-$header__border-color          : $gray !default;
-$header__font-size             : $font-size-small !default;
-$header__button-size           : 48px !default;
-$header__icon-size             : 24px !default;
-$header__icon-padding          : 2px !default;
-$header__icon-color            : $gray-dark !default;
+$header__margin--extra-large           : 0 !default;
+$header__font-size                     : $font-size-small !default;
+$header__border-color                  : $gray-light !default;
+
+$header__wrapper-padding               : 0 !default;
+$header__wrapper-padding--medium       : 12px 0 !default;
+
+$header__logo-padding                  : 12px 0 12px $spacer !default;
+$header__logo-padding--medium          : 0 0 0 $spacer !default;
+
+$header__menu-trigger-size             : 48px !default;
+
+$header__buttons-width                 : calc(100% - 48px)  !default;
+$header__buttons-width--medium         : calc(4 * 48px + 3 * #{$spacer}) !default;
+$header__buttons-width--extra-large    : calc(4 * 48px + 3 * #{$spacer--medium}) !default;
+
+$header__icon-size                     : 24px !default;
+$header__icon-padding                  : 2px !default;
+$header__icon-color                    : $gray-darker !default;
+
+$header__search-wrapper-padding        : 0 0 0 24px !default;
+$header__search-wrapper-padding--medium: 0 24px !default;
 
 .header {
     position: relative;
-    border-bottom: 1px solid $header__border-color;
+
+    &--menu-open {
+        overflow: visible;
```

Comment:
default value is visible? `.header` class (without modifier) shouldn't has `overflow: hidden`?

Reply:
I removed it, it left from FIlip's previous version.